test/minitest/test_minitest_mock.rb

```diff
@@ -227,17 +227,13 @@ def test_same_method_expects_are_verified_when_all_called
   end
 
   def test_same_method_expects_blow_up_when_not_all_called
-    mock = Minitest::Mock.new
-    mock.expect :foo, nil, [:bar]
-    mock.expect :foo, nil, [:baz]
-
-    mock.foo :bar
-
-    e = assert_raises(MockExpectationError) { mock.verify }
+    @mock = Minitest::Mock.new
+    @mock.expect :foo, nil, [:bar]
+    @mock.expect :foo, nil, [:baz]
```

Comment:
this chunk of diff (and more below) are needless diffs. Don't use ivars when you don't need to.